server/routes/api/representatives.js

```diff
@@ -129,6 +130,72 @@ router.get('/:zipCode', async (req, res) => {
   }
 })
 
+/*
+ * Returns string with the following properties:
+ *  - x-value: right or left. Indicates whether the photo should be cropped/positioned from the left or right side.
+ *  - y-value: top or bottom. Indicates whether the photo should be cropped/psoitioned from the top or bottom side.
+ *
+ * Example: "top left"
+ *
+ * @param {string} photoCropping - The photo cropping object from the CICERO API
+ * @returns {string} - The photo cropping CSS value
+ *
+ *
+ */
+function getPhotoCroppingValues(photo_cropping_object) {
+  // If the photo cropping object is not defined, return default value
+  if (!photo_cropping_object) {
+    return 'center center'
+  }
+
+  // args
+  console.log(photo_cropping_object)
+  let x = photo_cropping_object.x
+  let y = photo_cropping_object.y
+  let oriH = photo_cropping_object.oriHeight
+  let oriW = photo_cropping_object.origWidth
+
+  // 1. calculate threeshold for the x space
+  // we check if the coordinate starts on the left side of the image (the first half of the left side)
+  let x_left_threeshold = oriW / 2 / 2
+  // we check if the coordinate starts on the right side of the image (the first half of the right side) and we reduce a margin of 5% to be flexible
+  let x_right_threeshold = oriW / 2 - (5 / 100) * (oriW / 2)
```

Comment:
Extract the `(5 / 100)` into a variable so the margin can be adjusted for every calculation at the same time.  Also 0.05 would be a be a bit more readable for me, but that's just personal preference :)

Reply:
I do not have a preference about the 0.05. I agree about the value being more flexible with a variable.